scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export $(cat .env | xargs) &&
+if [[ -z "$NOTIFY_API_KEY" ]]; then
+    echo "NOTIFY_API_KEY must be provided" 1>&2
+    exit 1
+fi
+if [[ -z "$NOTIFY_TEST_TEMPLATE_ID" ]]; then
+    echo "NOTIFY_TEST_TEMPLATE must be provided" 1>&2
+    exit 1
+fi
```

Comment:
I don't think `NOTIFY_TEST_TEMPLATE_ID` should be required as we won't be setting it in all environments.

Reply:
Removed all references to the test template - it was just a standard template so not needed